src/app/modules/group/components/pending-request/pending-request.component.html

```diff
@@ -47,6 +47,16 @@
                 <p-sortIcon [field]="col.field"></p-sortIcon>
               </div>
             </th>
+            <th
```

Comment:
if `col.field === 'group.name'` you do the same as `at`? So just put it with "at".
But actually, the 3 columns are the same except the `width: 13.3333rem; `, so just use one code and use a condition for this width.

Reply:
Done

Reply:
much better